modules/cnvkit/1.0/envs/cnvkit-0.9.7.yaml

```diff
@@ -0,0 +1,120 @@
+name: null
```

Comment:
Can you please put this `.yaml` in the `lcr-modules/envs` and place here a symlink instead, just like for bcftools?

Reply:
Even for the module-specific yaml file?

Reply:
Yep, for consistency